src/components/menu/styledMenu.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+import { WorkWithUs } from '../sharedComponents/sharedComponents';
+
+export const Wrapper = styled.div`
```

Comment:
Try this and tell me what you think: remove the border radius, remove max-width, remove right positioning, change 'border' to 'border-bottom'

Reply:
I liked the old way better, but this is cleaner so we should go with it

Reply:
I think the main benefit of the way I recommended is as you resize the window, the menu text is always centered. Either way, we are taking liberties with the design so we can do whatever.